pyproject.toml

```diff
@@ -99,12 +99,17 @@ rosa = []
 #     Tornado 6.
 #     tornado
 # ]
+rose-edit = [
```

Comment:
(needs to match the `metomi-rose[edit]` below)

```suggestion
edit = [
```

Reply:
IMO @oliver-sanders I'd be happy for us to merge this.

Reply:
Done.